response.go

```
@@ -0,0 +1,44 @@
+}
+
+func (r BaseResponse) GetContent(c *gin.Context) any {
+	return r.Content
+}
+
+func (r BaseResponse) GetContentType(c *gin.Context) string {
+	return r.ContentType
+}
+
+func (r BaseResponse) SetCode(code int) {
+	r.Code = code
```

golangci-lint (1.64.8)

unusedwrite: unused write to field Code

(govet)